### PR TITLE
Patient characteristic tostring

### PR DESCRIPTION
**Pull request by losborne**
- [x] ! The fixtures in this PR were updated to reflect the behavior after https://github.com/projecttacoma/cqm-converter/pull/10 is merged, so this PR should be merged after that one.

This PR modifies the toString behavior of PatientCharacteristics to match frontend behavior.

In the frontend, we use constructor.name to extract the descriptions.  In the backend, we can look at _type.

Fixtures and tests were added as well.

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1626
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass

**Reviewer 1:**

Name: @edeyoung 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Reviewer 2:**

Name: @daco101 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: lib/helpers/results_helpers.js [hunk 1]
```diff
@@ -321,7 +321,7 @@ module.exports = class ResultsHelpers {
       }
       return quantityResult;
     } else if (result && result._type && result._type.includes('QDM::')) {
-      const description = result.description ? `${result.description}\n` : '';
+      const description = result.description ? `${result.description}\n` : `${result._type.replace('QDM::', '')}\n`;
```
Comment (edeyoung) [inline lib/helpers/results_helpers.js]: add comment. not clear to me why the `_type` is a good replacer for description when description not present.
